R/correlation.R

```diff
@@ -377,11 +393,19 @@ Correlation <- function(jaspResults, dataset, options){
       errors <- FALSE
       
       if(method != "pearson"){
-        result$conf.int <- .createNonparametricConfidenceIntervals(x = x, y = y, obsCor = result$estimate,
-                                                                   hypothesis = alternative, confLevel = conf.level,
-                                                                   method = method)
+        normal.approx.ci <- switch(method,
+                                   "spearman" = sample.size > 1290,
+                                   "kendall"  = sample.size > 100)
```

Comment:
Is the threshold 100 based on something particular? Could you please add a note on this in the help file?

Reply:
So in principle you don't need to do the asymptotic approximation do you? Can't you just grab them from the output of cor.test?